qa-tests-backend/src/test/groovy/AdmissionControllerTest.groovy

```diff
@@ -99,8 +106,17 @@ class AdmissionControllerTest extends BaseSpecification {
         // Wait for propagation to sensor
         sleep(10000 * (ClusterService.isOpenShift4() ? 4 : 1))
 
-        // Pre run scan to avoid timeouts with inline scans in the tests below
-        ImageService.scanImage(NGINX_IMAGE)
+        // Pre run scan to avoid registry timeouts with inline scans in the
+        // tests below.
+        ImageService.scanImage(SCAN_INLINE_IMAGE_NAME_WITH_SHA)
+
+        // Wait until we received metadata the inline scan image. This will
+        // ensure that enrichment has finalized.
+        withRetry(18, 10) {
```

Comment:
I do not think we need to retry things here, the `scanImage` call and scanning by Central/Scanner is done synchronously, i.e. once this has finished we can be sure that the image has finished scanning. So just one simple check for the image notes should be good enough.